README.markdown

```diff
@@ -1,181 +1,71 @@
 [![Build Status](https://travis-ci.org/ckan/ckanext-datapackager.png)](https://travis-ci.org/ckan/ckanext-datapackager) [![Coverage Status](https://coveralls.io/repos/ckan/ckanext-datapackager/badge.png?branch=master)](https://coveralls.io/r/ckan/ckanext-datapackager?branch=master)
 
-# Open Knowledge Data Packager
+# CKAN Data Packager
 
-The [Open Knowledge Data Packager](http://datapackager.okfn.org) is a web app for quickly creating and
-publishing [Tabular Data Packages](http://dataprotocols.org/tabular-data-package/).
-For more information, see the [blog post on CKAN.org](http://ckan.org/2014/06/09/the-open-knowledge-data-packager/)
-and the [about page on datapackager.okfn.org](http://datapackager.okfn.org/about).
+This extension adds importing and exporting of [Data Packages][data-packages] to [CKAN][ckan] datasets.
 
+## Requirements
 
-## Installing your own Data Packager
+* CKAN 2.4
 
-To install Data Packager:
+## Installing
 
-1. Get an Ubuntu 12.04, 64-bit server or virtual machine, ssh into it and
-   follow the steps below.
+To install `ckanext-datapackager` into a CKAN instance, do:
 
-2. Make sure Ubuntu's package index is up to date:
+1. If you're using a virtual environment for CKAN, activate it doing, for example:
 
-        sudo apt-get update
+        source /usr/lib/ckan/default/bin/activate
+    
+2. Install the `ckanext-datapackager` extension using pip:
 
-3. Install the Ubuntu packages that Data Packager requires:
+        pip install ckanext-datapackager
```

Comment:
Might be better to fix `setup.py` so `pip install ckanext-datapackager` installs its dependencies. Cloning the repository makes it harder for the users to use a specific version of the extension, and not simply following master.